services/viva-ms/lambdas/createVivaCase.js

```diff
@@ -165,18 +166,83 @@ async function putRecurringVivaCase(PK, workflowId, period) {
         period,
       },
       currentFormId: recurringFormId,
-      forms: prePopulatedForms,
     },
   };
 
-  const [putItemError, caseItem] = await to(putItem(putItemParams));
+  const casePersonList = getCasePersonList(vivaPerson);
+  caseItemPutParams.Item['persons'] = casePersonList;
+
+  const casePersonCoApplicant = getUserByRole(casePersonList, 'coApplicant');
+  if (casePersonCoApplicant) {
+    caseItemPutParams.Item['GSI1'] = `USER#${casePersonCoApplicant.personalNumber}`;
+  }
+
+  casePersonList.map(person => {
+    if (person.type === 'applicant') {
+      return { ...person, ...user };
```

Comment:
This will also not work as intended since this lambda triggers on bankid and we have no idea if the user that is logging in is the applicant. So we would need to check if person.personalNumber matches the user personalNumber instead and if so populate the person with user information :)